test/Microsoft.Azure.WebJobs.Host.FunctionalTests/DataBindingFunctionalTests.cs

```diff
@@ -51,6 +51,23 @@ public void BindNonStringableParameter_FailsIndexing()
                 exception.Message);
         }
 
+        [Fact]
+        public async Task FSharpTasks_AreCorrectlyIndexed()
+        {
+            // Arrange
+            MethodInfo method = typeof(FSharpFunctions.TestFunction).GetMethod("Run",
+                BindingFlags.Static | BindingFlags.Public);
+            Assert.NotNull(method); // Guard
+
+            FunctionIndexer indexer = FunctionIndexerFactory.Create(CloudStorageAccount.DevelopmentStorageAccount);
+            var indexCollector = new TestIndexCollector();
+            
+            // Act & Assert
+            await indexer.IndexMethodAsyncCore(method, indexCollector, CancellationToken.None);
+
+            Assert.NotEmpty(indexCollector.Functions);
```

Comment:
Can this be a stronger assert - the exact number you expect, name/parameters, etc.?